pkg/applier/applier.go

```diff
@@ -121,6 +122,7 @@ func (e ErrorEvent) Type() EventType {
 type Supervisor interface {
 	Applier
 	Destroyer
+	AnnotateStatusMode(ctx context.Context) error
```

Comment:
nit: UpdateStatusMode is probably slightly better, to avoid the caller needing to know it's an annotation. Tho it's a little weird since it doesn't take a new status mode. I guess it could, as long as it used the lock to avoid races.